client-spark/spark2/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -188,7 +188,7 @@ public RssShuffleManager(SparkConf sparkConf, boolean isDriver) {
     // External shuffle service is not supported when using remote shuffle service
     sparkConf.set("spark.shuffle.service.enabled", "false");
     LOG.info("Disable external shuffle service in RssShuffleManager.");
-    if (!sparkConf.getBoolean(RssSparkConfig.RSS_TEST_FLAG.key(), false)) {
+    if (!sparkConf.getBoolean(RssSparkConfig.RSS_TEST_MODE.key(), false)) {
```

Comment:
We shouldn't modify this place. It will cause that our spark2 integration tests fail.

Reply:
It's a little difficult to unify them. Because RSS_TEST_FLAG is used for ShuffleFlushManagerTest, RSS_TEST_MODE is used for integration tests and uts.

Reply:
They have different effect though they have similar names. For integraion tests, we need to set RSS_TEST_FLAG `false` value and set RSS_TEST_MODE `true` value.

Reply:
I see what you mean, give me some time to restore the original design.